examples/text_to_image/prompt.py

```diff
@@ -0,0 +1,202 @@
+list_of_prompts = [
+    "A futuristic cityscape with flying cars and neon lights",
```

Comment:
these prompts sound interesting. It would nice if the image follow these fantasies lol